app/views/endemics/date-of-visit.njk

```diff
@@ -21,8 +22,20 @@
           errorList: errorSummary
         }) }}
       {% endif %}
-     <h1 class="govuk-heading-l">Date of review or follow-up</h1>
-     <p>This is the date the vet last visited the farm for this review or follow-up. You can find it on the summary the vet gave you.</p>
+      <h1 class="govuk-heading-l">Date of 
+      {% if review %}
+        review 
+       {% else %}  
+         follow-up
+       {% endif %}
+      </h1>
+      <p>This is the date the vet last visited the farm for this
+      {% if review %}
+        review.
+      {% else %} 
+        follow-up.
+      {% endif %}
+        You can find it on the summary the vet gave you.</p>
```

Comment:
If you do this logic in the routes and pass the content into the template it might reduce the amount of if/else statements

Reply:
thanks for that